### ISSUE TITLE
dbcache config option is missing from example bitcoin.conf

### ISSUE THREAD

**Issue by lmj0011** [bitcoin/bitcoin#20947]
**Is your feature request related to a problem? Please describe.**

No, just requesting an update to the example bitcoin.conf

**Describe the solution you'd like**

`dbcache` config option to be added to [example config](https://github.com/bitcoin/bitcoin/blob/master/share/examples/bitcoin.conf)

ref: https://github.com/bitcoin/bitcoin/issues/20645#issuecomment-744078075

**Comment by FelixWeis** [comment]
An argument against could be made because `-dbcache` is mostly useful as a temporary configuration during IBD. After IBD the performance gain becomes more negligable. Argument for could be made as users may read sample config as a documentation instead of `-help`.

**Comment by lmj0011** [comment]
> An argument against could be made because -dbcache is **mostly useful as a temporary configuration during IBD**. After IBD the performance gain becomes more negligable. Argument for could be made as users may read sample config as a documentation instead of -help.

It wasn't so obvious to me that upping the `dbcache` would improve IBD based on the `-help` description. I propose that a better (newbie friendly) solution would be to prompt users on first run of IBD if they would like to increase `dbcache` and explaining the benefit of it. Of course, you may have to add a note to not increase cache past available system RAM to be completely fool-proof.


for reference, bitcoin-qt (v0.21.0) help description of `dbcache`
```shell
-dbcache=<n>
       Maximum database cache size <n> MiB (4 to 16384, default: 450). In
       addition, unused mempool memory is shared for this cache (see
       -maxmempool).
```

**Comment by fanquake** [comment]
> dbcache config option to be added to [example config](https://github.com/bitcoin/bitcoin/blob/master/share/examples/bitcoin.conf)

Should be solved after #22235.
